SimpleInventoryManagement/Util/IO.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+namespace SimpleInventoryManagement.Util
+{
+    /**
+     * this static class handle all input/output 
+     * operations, reading from user and logging (printing)
+     * messages on console.
+     */
+    public static class IO
+    {
+        /**
+         * used to print a message on console.
+         * take a message string and log type, to specify color of log.
+         * after printing, the console color reset.
+         */
+        public static void Log(string message, LogType type)
+        {
+            Console.ForegroundColor = (ConsoleColor)type;
+            Console.Write(message);
```

Comment:
@tjaber-fts 
I make it like that because I need the prompt message be at same line of user inputs.
but we can separate it into two methods: 
`Log()`  with new line. 
`LogPrompt()` without new line.
Is this solution good?